src/js/routes/about/About.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import AboutContent from '../../../static/about.html';
+
+const About = () => (
+	<div>
+		<div dangerouslySetInnerHTML={{ __html: AboutContent }} />
```

Comment:
I'd have just put the content of `about.html` directly here and deleted the file `about.html`. Might have made this a [pure component](https://facebook.github.io/react/docs/react-api.html#react.purecomponent) and utilised it in some fashion, though frankly I don't think that that is necessary.

I'm no expert in this area (that is routing), so feel free to educate if the above is naive. I may also be stuck in React Native mode ;).